src/common/paths/ping.yml

```
@@ -1,7 +1,13 @@
 get:
   operationId: GetPing
-  summary: Get the status and version of the instance
-  description: Returns the status and InfluxDB version of the instance.
+  summary: Get the status of the instance
+  description: |
+    Returns the status and InfluxDB version of the instance.
+
+    Use this endpoint to monitor instance uptime.
```

```suggestion
    Use this endpoint to monitor uptime for the InfluxDB instance.
```

Link to an example task, related guide if we have one?